coderd/database/dbtestutil/coder_testing.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS test_databases (
+	name text PRIMARY KEY,
+	created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	dropped_at timestamp with time zone, -- null means it hasn't been dropped
+	process_uuid uuid NOT NULL,
+	-- these are both unused for now, but we'd like to include them for statistics later
+	test_package text,
+	test_name text
```

Comment:
We should remove these for now, and add them back when we actually use them with `ALTER TABLE test_databases ADD COLUMN IF NOT EXISTS` to demonstrate how to add columns safely.